README.md

```diff
@@ -176,21 +176,21 @@ Run this command:
 
 You will also need to run this command whenever ``requirements.txt`` or ``test_requirements.txt`` change.
 
-#### 4) Create database tables
-To do this, run this command:
+#### 4) Create initial data
+First, create the database tables from the Django models:
 
     docker-compose run web ./manage.py migrate
 
-#### 5) Create an Elasticsearch index
-To do this, run this command:
+Then, populate example data into these tables, for development purposes:
 
-    docker-compose run web ./manage.py recreate_index
```

Comment:
I don't think we should remove this step. `recreate_index` is essential for deploying the app, and production instances will not call `seed_db` beforehand